### PR TITLE
trio-aiohttp

### PR DESCRIPTION
**Pull request by scopatz**
noarch: python

Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - this is ready for review!

File: recipes/trio-aiohttp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5.0
+    - pip
+    - trio-asyncio
+    - aiohttp
```
Comment (jakirkham) [inline recipes/trio-aiohttp/meta.yaml]: Could we drop everything not used to build here?

Comment (scopatz) [reply recipes/trio-aiohttp/meta.yaml]: The setup.py has `install_requires` with these, so I think that they are needed.

Comment (jakirkham) [reply recipes/trio-aiohttp/meta.yaml]: So that makes sense for having them in `run`, but we usually only add things from `setup_requires` in `host`.

Comment (scopatz) [reply recipes/trio-aiohttp/meta.yaml]: Ahh right, I was getting confused with another package.

Comment (jakirkham) [reply recipes/trio-aiohttp/meta.yaml]: No worries 🙂

**Comment by jakirkham** [comment]
Thanks @scopatz! 😄

**Pull request merged**